src/components/Person/Person.jsx

```diff
@@ -1 +1,11 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {`${person.partnerName} is my ${person.sex === 'f' ? 'husband' : 'wife'}`}
+      </p>
+    ) : <p className="Person__partner">I am not married</p>}
```

Comment:
You could consider deconstructing person for clarity, e.g.:
` const { name, age, isMarried, sex, partnerName } = person;`

Reply:
I would say that in this case it's just a matter of preference. I use destructuring on objects only when I want to access some value more than once... or when it simply looks better ;)